.github/workflows/ci-nym-vpn-core.yml

```diff
@@ -120,6 +130,12 @@ jobs:
         run: |
           ./wireguard/build-wireguard-go.sh
 
+      - name: Build wireguard (iOS)
+        if: contains(matrix.os, 'macos') || contains(matrix.os, 'mac-m1')
+        shell: bash
+        run: |
+          ./wireguard/build-wireguard-go.sh --ios
```

Comment:
In the build script it seems like ios and macos are different code paths, but here this command is applied for both?

Reply:
This is a mistake. Let me fix this

Reply:
Removed

Reply:
Actually. This is correct because we build wireguard-go for 3 platforms here:

- Desktop
- Android
- iOS (added in this PR)

Reply:
Yep looks good!